### PR TITLE
Some suggested sequencing type simplifications

### PR DESCRIPTION
**Pull request by illusional**
Use the new `sequencing_group.sequencing_type` attribute rather than fetching from the config.
These are just soft, take-it-or-leave-it suggestions, I'm not personally tied to them, just thought it would be interesting to see what it might look like.

Stacked on #851, but I'd advise against merging into there, and I'd just rebase when that gets merged if we're interested in these changes.

### CODE REVIEW

File: cpg_workflows/large_cohort/combiner.py [hunk 1]
```diff
@@ -70,6 +69,14 @@ def run(out_vds_path: Path, tmp_prefix: Path, *sequencing_group_ids) -> hl.vds.V
         sequencing_groups = [s for s in sequencing_groups if s in sequencing_group_ids]
 
     sequencing_groups = _check_gvcfs(sequencing_groups)
+    seq_types = set(s.sequencing_type for s in sequencing_groups)
+    if len(seq_types) > 1:
+        sequencing_type = config_retrieve(['workflow', 'sequencing_type'])
+        logging.warning(
+            f'Found multiple sequencing types for large cohort combiner: {seq_types}, using the config-provided {sequencing_type}',
+        )
```
Comment (MattWellie) [inline cpg_workflows/large_cohort/combiner.py:77]: this one is spicy, for now I reckon this should be a log/fail with a ValueError. There might be some situations where we would mix sequencing types but probably not this one

**Review by MattWellie — COMMENTED** [review]
In general it's logical. For genotype/align when they're split out as per-SG workflows we could run everything, and each SG would have the appropriate jobs queued according to its type.
Iffy feelings about the combiner one though, that feels like cause for failure at the moment

**Review by MattWellie — APPROVED** [review]
LGTMatt

File: cpg_workflows/large_cohort/combiner.py (hunk 1 above)
Comment (illusional) [reply cpg_workflows/large_cohort/combiner.py]: Yeah agreed, upgraded to an error

**Comment by illusional** [comment]
Hey @MattWellie, do you want to merge this PR - otherwise I might close.

**Comment by MattWellie** [comment]
> Hey @MattWellie, do you want to merge this PR - otherwise I might close.

I like it, just gonna smash merge on it. I was waiting for @vivbak to review as well, but I think this is straightforward enough

**Pull request merged**